friendly-captcha/includes/admin.php

```diff
@@ -53,4 +53,33 @@ function frcaptcha_admin_notice__not_configured() {
 
         add_action( 'admin_notices', 'frcaptcha_admin_notice__not_configured' );
     }
+
+    if (isset($_GET['frcaptcha-verification-failed-dismissed'])) {
+        FriendlyCaptcha_Plugin::$instance->remove_verification_failed_alert();
+    }
+
+    if (FriendlyCaptcha_Plugin::$instance->get_verification_failed_alert()) {
+        function frcaptcha_admin_notice__verification_failed() {
+            $url = esc_url( add_query_arg(
+                'page',
+                'friendly_captcha_admin',
+                get_admin_url() . 'options-general.php'
+            ) );
+    
+            ?>
+            <div class="notice notice-error is-dismissible">
+                <p>
+                    <b>Friendly Captcha verification has failed!</b> 
+                    <br>
+                    This is usually because you have entered an incorrect API Key. Please visit the <a href="<?php echo $url ?>">Friendly Captcha settings</a> and enter a valid Sitekey and API Key.
+                </p>
```

Comment:
Would it be possible to add something like the error or the raw body of the request? I'm not sure how to best persist it 🤔

Reply:
It now shaws the error response as JSON:
![grafik](https://github.com/FriendlyCaptcha/friendly-captcha-wordpress/assets/33966852/c937b30c-fb0a-4557-82a4-d376e15deea8)